config/form_profile_mappings/22-1990EZ.yml

```diff
@@ -0,0 +1,10 @@
+veteranFullName: [identity_information, full_name]
+gender: [identity_information, gender]
+veteranDateOfBirth: [identity_information, date_of_birth]
+veteranSocialSecurityNumber: [identity_information, ssn]
+homePhone: [contact_information, us_phone]
+mobilePhone: [contact_information, mobile_phone]
+veteranAddress: [contact_information, address]
+email: [contact_information, email]
+toursOfDuty: [military_information, tours_of_duty]
+currentlyActiveDuty: [military_information, currently_active_duty_hash]
```

Comment:
I'm not sure what this maps to but this is the config settings for another 1990 form